var/spack/repos/builtin/packages/lbann/package.py

```diff
@@ -19,7 +19,8 @@ class Lbann(CMakePackage, CudaPackage):
 
     maintainers = ['bvanessen']
 
-    version('develop', branch='develop')
+    version('develop', preferred=True, branch='develop')
```

Comment:
I'm hesitant to make a non-checksummable moving target the version we install by default...

Reply:
Yeah, I did that because the way that the concretizer resolves the default dependencies and fails at backtracking means that a command like `spack install lbann` will fail because dependent libraries like Hydrogen have a default that is incompatible with the last released version of LBANN.  So a user has to do something like `spack install lbann@0.101` to get the proper behavior, even though that is the default value.